### DISCUSSION TITLE
Vercel not generating SSL Certificate

### DISCUSSION THREAD

**Discussion by RamyBouchareb25** [vercel/community#6508] [Help]
### Summary

So i have deployed my next app on vercel and am now trying to add my domain name to it
but when i added it it couldn't generate the ssl certificate stuck on `Generating ssl certificate` not sure what to do 
i looked it up and found that i had to add the CAA record `0 issuer letsencrypt.org`added it waited a few hours and stil no ssl certificate 

so if anyone can help me find the reason

### Steps to Reproduce

My domain name is shop.ramybouchareb.me 
*PS : I am using a subdomain here my domain name of ramybouchareb.me is being used on github pages not sure if it's relevent*

**Comment by swarnava** [comment]
Hi,

It looks like there is a DNS issue with your domain: https://letsdebug.net/shop.ramybouchareb.me/1875324

Here is the error we are receiving from Letsencrypt:

```We could not generate a cert for shop.ramybouchareb.me because the required http-01 challenge failed. DNS problem: looking up A for shop.ramybouchareb.me: DNSSEC: DNSKEY Missing; DNS problem: looking up AAAA for shop.ramybouchareb.me: DNSSEC: DNSKEY Missing```
